scss/main.scss

```diff
@@ -765,6 +769,11 @@ $animation-length: 80s;
     z-index: 1;
     margin: 0px 50px;
 }
+
+.sponsor-img.smaller {
+    width: 500px;
+}
```

Comment:
oops haha sorry wasnt clear about this - can you resize the actual image that we use under the img folder? so that way when ppl load the site the file size of those images is reduced

Reply:
Done!